### PR TITLE
Add cleanUrl paramater in the supported sites

### PR DESCRIPTION
**Pull request by hemantapkh**
Hi there, I wanted to shorten a magnet link that starts with magnet:? but the clean_url method is adding http or https before the link. So, I added a cleanUrl argument that defines whether to clean the URL or not. 

I checked all the supported sites by pyshorteners and only Tinyurl.com & is.gd allows link without http or https. So, I added the cleanUrl argument in those sites only.

By default, cleanUrl is True so it won't break any existing projects.😇

### CODE REVIEW

**Review by ellisonleao — CHANGES_REQUESTED** [review]
Can you switch that logic to the main method [here](https://github.com/ellisonleao/pyshorteners/blob/main/pyshorteners/base.py#L141), please ?

also, can you keep the `snake_case` convention?

**Comment by hemantapkh** [comment]
I am not sure about switching the logic to the main method. Can you elaborate more or edit it yourself? @ellisonleao

**Comment by ellisonleao** [comment]
> I am not sure about switching the logic to the main method. Can you elaborate more or edit it yourself? @ellisonleao

Basically all shorteners are inheriting methods from the `BaseShortener` class. If you change the `clean_url` method in the `BaseShortener`, it will reflect on all other shortener classes.  Does that makes sense?

**Comment by hemantapkh** [comment]
Done, Now I can change this behavior while creating the object 

> > I am not sure about switching the logic to the main method. Can you elaborate more or edit it yourself? @ellisonleao
> 
> Basically all shorteners are inheriting methods from the `BaseShortener` class. If you change the `clean_url` method in the `BaseShortener`, it will reflect on all other shortener classes. Does that makes sense?

Done, now I can change this behavior while creating the object like this:

`s = pyshorteners.Shortener(clean=False)`

Is this what you said?

And why are the _get and _post methods cleaning the api_url [here](https://github.com/ellisonleao/pyshorteners/blob/5d8e6cda6742f0b6b752e0ce576bb7901119c118/pyshorteners/base.py#L63)? Isn't the api_url already cleaned? Or I misunderstood?

File: pyshorteners/base.py [hunk 1]
```diff
@@ -42,6 +42,7 @@ def __init__(self, **kwargs):
         self.verify = getattr(self, "verify", True)
         self.proxies = getattr(self, "proxies", {})
         self.cert = getattr(self, "cert", None)
+        self.clean = getattr(self, "clean", True)
```
Comment (ellisonleao) [inline pyshorteners/base.py:45]: this should a parameter in the function and not a class attribute. Can you move that to the function?

Comment (hemantapkh) [reply pyshorteners/base.py]: Sorry for the delay. Is it okay now? @ellisonleao

**Pull request closed without merging**